src/app/qgscustomization.cpp

```diff
@@ -501,7 +503,7 @@ QString QgsCustomizationDialog::widgetPath( QWidget *widget, const QString &path
 
   QString pathCopy = path;
 
-  if ( !QgsCustomization::sInternalWidgets.contains( name ) )
+  if ( !sInternalWidgets()->contains( name ) )
```

Comment:
This one could also be easily avoided, by:

- making a private function "isInternalWidget"
- using a local static in isInternalWidget